### DISCUSSION TITLE
What's the difference between these two SUSE12 distro?

### DISCUSSION THREAD

**Discussion by zhufeizzz** [microsoft/WSL#7918] [Q&A]
![双版本2](https://user-images.githubusercontent.com/4591255/148643704-6cac992a-a6b6-4ea3-9d08-6b92e5351aa0.PNG)

The first one I got from here.
https://docs.microsoft.com/zh-cn/windows/wsl/install-manual#downloading-distributions
https://aka.ms/wsl-sles-12

The second one is on the Windows Store.

What's the difference? I want to install SUSE12 offline, how can I download the same version with Windows Store?

**Comment by herrwinfried** [comment]
I know you can't update if you download from outside the Microsoft page.

**Comment by zhufeizzz** [comment]
> I know you can't update if you download from outside the Microsoft page.

Actually I can  update it from 1.0.2.0 to 1.1.0.0 as the picture shows.
